story.txt

```diff
@@ -4,3 +4,5 @@ Hiroki asked me a question about how to open previous changes.
 
 I completely ignored Hiroki, and decided to buy a fish
 a crsipy fish to slap Ashbin in the hamstrangs.
+
+// Hiroki changed 2021 01 09
```

Comment:
This is just empty lines. Please add content to the story.

Reply:
@ashby-git fixed 2020 01 10

Reply:
:(